farm/modeling/adaptive_model.py

```diff
@@ -271,6 +274,7 @@ def connect_heads_with_processor(self, tasks, require_labels=True):
             else:
                 num_labels = len(label_list)
             head.resize_output(num_labels)
+            head.to(self.device)
```

Comment:
Wasn't that already part of a previous PR?

Reply:
Yes this was not the cleanest way to introduce this change into the branch